Первое задание для Library выполнено

PopovMatvey opened the pull request:
Свёрстан header и footer в соответствии с макетом и требованиями.

PopovMatvey committed:
Create style.css

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Create avatar-01.png

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Delete Proxima Nova Regular.otf

PopovMatvey committed:
Первое задание для Library выполнено

Свёрстан header и footer в соответствии с макетом и требованиями.

djilua1 commented on src/library_task/index.html:
```diff
@@ -1,12 +1,73 @@
+  </footer>
+</body>
+
+</html>
```
![image](https://user-images.githubusercontent.com/43634050/164260791-e3cbd7c6-c26d-4658-bc76-c74433b0d710.png)
Лишние паддинги и футер не прибит к низу.

djilua1 commented on src/library_task/index.html:
```diff
@@ -1,12 +1,73 @@
+    <div class="menu-field d-flex align-items-center">
+      <ul class="menu-field_list d-flex">
+        <li>
+          <a class="menu-field_list__first_item" href="#">Help Center</a>
```
Это больше на кнопку похоже.

djilua1 commented on src/library_task/index.html:
```diff
@@ -1,12 +1,73 @@
+              <path fill-rule="evenodd" fill-opacity="0.749" fill="rgb(25, 32, 46)"
+                d="M2.000,-0.000 C3.104,-0.000 4.000,0.895 4.000,2.000 C4.000,3.105 3.104,4.000 2.000,4.000 C0.895,4.000 -0.000,3.105 -0.000,2.000 C-0.000,0.895 0.895,-0.000 2.000,-0.000 Z" />
+            </svg>
+          </span>
```
Кнопка.

djilua1 commented on src/library_task/index.html:
```diff
@@ -1,12 +1,73 @@
+      </div>
+    </div>
+
+  </footer>
```
Тут тоже везде кнопочки.

djilua1 commented on src/library_task/assets/css/style.css:
```diff
@@ -0,0 +1,127 @@
+    justify-content: center;
+}
+
+.align-items-center {
```
Эти 3 класса странно наименование имеют.

djilua1 commented on src/library_task/assets/css/style.css:
```diff
@@ -0,0 +1,127 @@
+
+.menu-field_list__second_item {
+    display: block;
+    margin: -1px 15px 0px 15px;
```
Отрицательные марджины плохая практика.

djilua1 commented on src/library_task/assets/css/style.css:
```diff
@@ -0,0 +1,127 @@
+    height: 45px;
+    width: 55px;
+    border-radius: 0px 0px 5px 0px;
+}
```
Поменьше рандомных чисел. Все это обычно дорабатывается марджинами или падингами от элементов, остальное флексы решают.

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Add files via upload

Были исправлены все семь замечаний:
1. Были убраны лишние отступы в body, footer теперь прибит к низу страницы;
2, 3, 4. Все символы в вёрстке были переделаны под кнопки;
5. Три класса: "d-block", "justify-content-center", "align-items-center"  были переделаны в один: "wrapper";
6. Отрицательное значение в свойстве margin было изменено;
7. Для полей с кнопками в footer свойства heigt и width были изменены на padding.
